spec/requests/users_spec.rb

```diff
@@ -56,4 +56,44 @@
       end
     end
   end
+
+  describe 'GET /users/edit' do
+    let!(:user) { create(:user) }
+
+    it 'アクセスに成功する' do
+      aggregate_failures do
+        get edit_user_path(user.id)
+        expect(response).to have_http_status(:success)
+        expect(response.body).to include 'ユーザー情報編集'
+      end
+    end
+  end
+
+  describe 'PUT /users' do
+    let!(:user) { create(:user) }
+
+    context '有効なパラメータのとき' do
+      it 'ユーザーを更新する' do
+        aggregate_failures do
+          put user_path(user.id), params: { user: { email: 'new_email@example.com' } }
+          user.reload
+          expect(user.email).to eq('new_email@example.com')
+          expect(response).to redirect_to(user)
+        end
+      end
+    end
+
+    let!(:user_email) { user.email }
+
+    context '無効なパラメータのとき' do
+      it 'ユーザーを更新しない' do
+        aggregate_failures do
+          post user_path(user.id), params: { user: { email: nil } }
```

Comment:
Idomatic

user_emailはショートハンドとして使いたいだけなら、これは不要だと思う。

```suggestion
    context '無効なパラメータのとき' do
      let!(:params) {  { user: { email: nil } } }

      it 'ユーザーを更新しない' do
        aggregate_failures do
          post user_path(user.id), params: params
```

Reply:
[3029a1a](https://github.com/ok-os-job-change-team/tetsuya-twitter-clone-bootcamp/pull/36/commits/3029a1a61bab4a305274757367b1c806d74260bc) で対応しました。